src/components/LicenseForm/LicenseForm.js

```diff
@@ -168,7 +168,7 @@ class LicenseForm extends React.Component {
                           />
                         </Col>
                       </Row>
-                      <LicenseFormInfo {...this.getSectionProps('licenseFormInfo')} />
+                      <LicenseFormInfo {...this.getSectionProps('licenseFormInfo')} change={this.props.change} />
```

Comment:
This seems kludgey. May as well give all the license form sections `change` so it's not confusing why one has it and another doesn't, no?